presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java

```diff
@@ -117,4 +123,14 @@ public static HiveStorageFormat getHiveStorageFormat(Map<String, Object> tablePr
     {
         return (List<String>) tableProperties.get(BUCKETED_BY_PROPERTY);
     }
+
+    public static String getFieldDelimProperty(Map<String, Object> tableProperties)
+    {
+        return StringEscapeUtils.unescapeJava((String) tableProperties.get(FIELD_DELIM_PROPERTY));
```

Comment:
We shouldn't have a special escape format for strings. Users can specify this in SQL:
```sql
CREATE TABLE ... WITH (field_delimiter = chr(1))
```
For printing in the SQL formatter, we should should detect when a string literal contains non-printable characters and switch to the escaped Unicode format (which needs to be implemented in Presto): https://github.com/prestodb/presto/issues/6585